corehq/apps/export/models/new.py

```diff
@@ -211,6 +232,14 @@ def generate_instance_from_schema(schema, domain, app_id=None):
         return instance
 
 
+class CaseExportInstance(ExportInstance):
+    case_type = StringProperty()
+
+
+class FormExportInstance(ExportInstance):
+    xmlns = StringProperty()
+
+
```

Comment:
`CaseExportInstance` and `FormExportInstance` are a little redundant with the `type` property. Maybe we should check the `doc_type` instead of using `type`?